hedera-node/hedera-util-service-impl/src/test/java/com/hedera/node/app/service/util/impl/test/handlers/UtilPrngHandlerTest.java

```diff
@@ -30,10 +48,215 @@ class UtilPrngHandlerTest {
     @Mock
     private PreHandleContext preHandleContext;
 
-    private UtilPrngHandler subject = new UtilPrngHandler();
+    @Mock
+    private HandleContext handleContext;
+
+    @Mock
+    private ReadableRunningHashLeafStore readableRunningHashLeafStore;
+
+    private UtilPrngHandler subject;
+    private UtilPrngTransactionBody txn;
+    private PrngRecordBuilder recordBuilder;
+    private static final Hash hash = new Hash(Utils.randomUtf8Bytes(48));
+    private static final RunningHash nMinusThreeHash = new RunningHash(hash);
+
+    private static final PrngConfig config = new PrngConfig(true);
+
+    @BeforeEach
+    void setUp() {
+        subject = new UtilPrngHandler();
+        recordBuilder = new UtilPrngRecordBuilder();
+        givenTxnWithoutRange();
+    }
+
+    @Test
+    void preHandleValidatesRange() {
+        final var body = TransactionBody.newBuilder()
+                .utilPrng(UtilPrngTransactionBody.newBuilder())
+                .build();
+        given(preHandleContext.body()).willReturn(body);
+        assertDoesNotThrow(() -> subject.preHandle(preHandleContext));
```

Comment:
I believe we agreed to use AssertJ in new tests.

Reply:
Changed to use AssertJ